index.test.js

```diff
@@ -236,4 +236,28 @@ describe('Success', function() {
     const expected = await strToQuads(fs.readFileSync('./test/tc01/output.nq', 'utf-8'));
     assert.ok(isomorphic(result.output, expected));
   });
+
+  it('Configure function state ID', async () => {
+    const stateFolder = tempFolderPath + '/fstate';
+    const wrapper = new RMLMapperWrapper(
+        rmlmapperPath,
+        tempFolderPath,
+        false,
+        stateFolder,
+        300);
+
+    const rml = fs.readFileSync('./test/functionstate/mapping.ttl', 'utf-8');
+    const sources = {
+      'student.csv': fs.readFileSync('./test/functionstate/student.csv', 'utf-8')
+    };
+
+    await wrapper.execute(rml, {sources, generateMetadata: false, asQuads: true});
```

Comment:
How do we know if the state as actually kept? Don't we need to do at least two executions?

Reply:
God point. This just tests if state is kept, not if it is used the next time. I'll add that.

Reply:
Test extended. Could you review again?